### PR TITLE
Embed timing diagram within job details page

### PR DESCRIPTION
**Pull request by rsasch**
- uses ng2-google-charts library
- does not use `/timing` endpoint, instead is based on metadata that is already there

Closes: #355

### CODE REVIEW

File: ui/src/app/job-details/tabs/timing-diagram/timing-diagram.component.ts [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+  buildTimelineData(metadata: TaskMetadata[]): void {
+    this.timelineChart = {
+      chartType: 'Timeline',
+      dataTable: []
+    };
+    this.timelineChart.dataTable.push(['Num', 'Name', 'Start', 'End']);
+    let counter = 1;
+
+    metadata.forEach((task) => {
+      this.timelineChart.dataTable.push([counter.toString(), task.name, task.start, task.end]);
```
Comment (calbach) [inline ui/src/app/job-details/tabs/timing-diagram/timing-diagram.component.ts]: Are these guaranteed to already be in a reasonable sorted order?

File: ui/src/app/job-details/tabs/tabs.component.html [hunk 2]
```diff
@@ -128,13 +128,9 @@
     <mat-tab *ngIf="hasOutputs()" label="Outputs">
       <jm-resources-table [entries]="job.outputs"></jm-resources-table>
     </mat-tab>
-    <!-- Disable the graph view tab so the indicator doesn't switch on click -->
-    <mat-tab *ngIf="hasTimingUrl()" label="Graph View" [disabled]=true>
-      <ng-template mat-tab-label>
-        <a style="all: unset" href="{{ job.extensions.timingUrl }}" target="_blank">
```
Comment (cjllanwarne) [inline ui/src/app/job-details/tabs/tabs.component.html]: Can we now remove this extension from the Cromwell shim code?

Comment (rsasch) [reply ui/src/app/job-details/tabs/tabs.component.html]: Good point – removed.

File: ui/src/app/job-details/tabs/timing-diagram/timing-diagram.component.ts (hunk 1 above)
Comment (rsasch) [reply ui/src/app/job-details/tabs/timing-diagram/timing-diagram.component.ts]: I believe https://github.com/DataBiosphere/job-manager/blob/066fc41b6a4e885c058fbd5da42688d5d6662caf/servers/cromwell/jobs/controllers/jobs_controller.py#L112 sorts the tasks by start time.

**Pull request merged**